containerd-shim-v2/service.go

```diff
@@ -274,7 +298,20 @@ func getTopic(e interface{}) string {
 	return cdruntime.TaskUnknownTopic
 }
 
+func trace(ctx context.Context, name string) (opentracing.Span, context.Context) {
+	if ctx == nil {
+		logrus.WithField("type", "bug").Error("trace called before context set")
+		ctx = context.Background()
+	}
+	span, ctx := opentracing.StartSpanFromContext(ctx, name)
+	span.SetTag("subsystem", "runtime")
```

Comment:
The `Trace()` function sets the `source` tag to `runtime`, not the `subsystem` one. What is the reason for the difference?

Reply:
The `Trace()` function has an additional `created span` message, which according to the comment, is essential for tracing testing. Thag begs the question: how do you plan to test this newly added tracing?

Reply:
@c3d I switched it to `source`, it should be consistent.

Regarding the `created span` message, the test is undergoing significant changes in a PR (https://github.com/kata-containers/tests/pull/2129) and will not check for that string.